lib/include/fdotypes.h

```diff
@@ -237,37 +237,63 @@ bool fdo_public_key_write(fdow_t *fdow, fdo_public_key_t *pk);
 fdo_public_key_t *fdo_public_key_read(fdor_t *fdor);
 fdo_public_key_t *fdo_public_key_clone(fdo_public_key_t *pk);
 
-#define AES_IV 16
-#define AES_CTR_IV 12
-#define AES_CTR_IV_COUNTER 4
+// AES GCM IV length
+#define AES_GCM_IV 12
```

Comment:
The AES_GCM_xxx definitions are required only if AES_MODE_GCM_ENABLED is defined. We can consider moving these definitions inside the block defined below in line 250. The generic definitions can be kept outside the if block.